processor/k8sprocessor/config.go

```diff
@@ -193,3 +197,13 @@ type PodAssociationConfig struct {
 	// e.g. ip, pod_uid, k8s.pod.ip
 	Name string `mapstructure:"name"`
 }
+
+// PodExcludeConfig represent a list of Pods to exclude
+type PodExcludeConfig struct {
```

Comment:
I'd name this just `ExcludeConfig` as we could support excluding things other than pods in future.